content/engineering/tests.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Automated Testing
+sidenav: true
+permalink: /engineering/tests/
+tags: engineering
+layout: layouts/page
+eleventyNavigation: 
+  parent: engineering_tools
+  key: Automated Testing
+  order: 6
+  title: Automated Testing
+---
+
+There are many different types of automated testing, which all have
+separate roles to play. Ideally, automated testing can be run locally
+as well as part of a continuous integration workflow. 
+## Validating HTML output
+
+### Ruby {#html-output-ruby}
```

Comment:
Not sure why this has a custom anchor (currently not working until we add MarkdownItAnchor), wondering if we can remove. We can search for references to `#html-output-ruby`